qa/rpc-tests/wallet_orchard_change.py

```diff
@@ -65,17 +70,22 @@ def run_test(self):
         acct1 = self.nodes[1].z_getnewaccount()['account']
         ua1 = self.nodes[1].z_getaddressforaccount(acct1, ['orchard'])['address']
 
-        recipients = [{"address": ua1, "amount": 1}]
-        myopid = self.nodes[0].z_sendmany(ua0, recipients, 1, 0, 'AllowRevealedAmounts')
+        recipients = [{"address": ua1, "amount": Decimal('1')}]
+        # TODO The z_sendmany call fails when passed `null` because it calculates a fee
+        # that is too low. If we passed `fee` in here instead, it would succeed.
```

Comment:
```suggestion
        # TODO The z_sendmany call fails when passed `None`/`ZIP_317_FEE` because it
        # calculates a fee that is too low. If we pass `fee` instead of `None`, it succeeds.
```